src/main/features/response/draft/responseDraft.ts

```diff
@@ -106,10 +114,24 @@ export class ResponseDraft extends DraftDocument {
       && this.rejectPartOfClaim.option === option
   }
 
+  public requirePartialAdmitHandOff (): boolean {
+    if (this.isResponsePopulated() && this.response.type === ResponseType.PART_ADMISSION) {
+      return false
+    }
+  }
+
   public requireMediation (): boolean {
     return this.isResponsePopulated() && (this.isResponseRejectedFully() || this.isResponseRejectedPartially())
   }
 
+  public requireWhenDidYouPay (): boolean {
```

Comment:
It is not draft responsibility to say what is required or not. Task list is responsible for answering question what else is needed to complete application judging by current state of the draft.

Remove from draft class.